app/views/warehouse_reports/health/agency_performance/_report_download.haml

```diff
@@ -0,0 +1,14 @@
+:ruby
+  if export.present?
+    data = {
+      type: export.type,
+      query_string: export.query_string
+    }
```

Comment:
It feels like this should be moved into the export class so you can call `export.link_data` or something similar.